### PR TITLE
CRM457-2188: Remove previous mentions to eforms

### PR DESCRIPTION
**Pull request by elken**
## Description of change

Remove all textual and error mentions to eforms

[Link to relevant ticket](https://dsdmoj.atlassian.net/browse/CRM457-2188)

### CODE REVIEW

File: config/locales/en/errors.yml [hunk 1]
```diff
@@ -47,14 +47,6 @@ en:
         heading: Access restricted
         lead_text: This page requires you to be signed in to LAA Portal. You will be redirected back to this service after you sign in.
         sign_in_button: Sign in with LAA Portal
-      not_enrolled:
```
Comment (jogilder) [inline config/locales/en/errors.yml:50]: I don't think you should take all of this out - there is still the issue of someone giving them the link and they don't have the correct role from portal.

File: gems/laa_multi_step_forms/app/controllers/laa_multi_step_forms/errors_controller.rb [hunk 2]
```diff
@@ -19,10 +19,6 @@ def unauthorized
       respond_with_status(:unauthorized)
     end
 
-    def not_enrolled
```
Comment (jogilder) [inline gems/laa_multi_step_forms/app/controllers/laa_multi_step_forms/errors_controller.rb:22]: this should stay

**Review by jogilder — COMMENTED** [review]
not enrolled is still required I believe

File: config/locales/en/errors.yml (hunk 1 above)
Comment (elken) [reply config/locales/en/errors.yml:50]: Oh I see, I thought that was for something else. Will amend

**Review by robinsk1 — APPROVED** [review]

**Pull request merged**